spec/game_loader_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'yaml'
+require 'game_loader'
+require_relative './test_doubles/fake_class_double'
+
+RSpec.describe GameLoader do
+  describe 'Loading an object' do
```

Comment:
You could be more specific - `an object` doesn't tell much to a reader.

Reply:
I agree. I'll change this